src/app.js

```diff
@@ -1,8 +1,13 @@
 import express from 'express'
+import fallback from 'express-history-api-fallback'
+import path from 'path'
 import {meta, time_seriesDB, heatmapDB} from './util/initDB.js'
 
 const app = express()
-app.use(express.static('public'))
+const root = path.join(__dirname, '../public')
+console.log(root)
+app.use(express.static(root))
+app.use(fallback('index.html', { root }))
```

Comment:
@cbas I'm trying to use `express-history-api-fallback` to fallback to `index.html`, but line 10 is causing the following error `Uncaught (in promise) SyntaxError: Unexpected token <`

Could you help please?

cc @yongjun21